build.py

```diff
@@ -1179,6 +1181,15 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
         df += fastertransformer_buildscript.create_postbuild(
             is_multistage_build=False)
 
+    if target_machine == "s390x":
```

Comment:
@nv-kmcgill53 this is mainly for E2E testing as we do not publish such an image. Might be a better way to handle this however.

Reply:
I need to think more about this. Thanks for pointing this out.

Reply:
I think with @Tabrizian suggestion of using the standard image - we can remove this. The remaining bit would be the miniconda version - but we could also provide the ability to simply exclude miniconda